config/config.php

```diff
@@ -24,11 +24,16 @@
  */
 $install_dir = env('V1_INSTALL_DIR', '/opt/librenms');
 $config_source = $install_dir . '/config.php';
+$defaults_source = $install_dir . '/includes/defaults.inc.php';
 
 $config = array();
 
 if (file_exists($config_source)) {
     include_once($config_source);
 }
 
+if (file_exists($defaults_source)) {
```

Comment:
I actually didn't mean for this to go in really but for now I think we should. It will take a while to move everything over to use the DB.